### PR TITLE
restructure section 'documentation'

### PR DESCRIPTION
**Pull request by f-brinkmann**
account for commentrs in #98

### CODE REVIEW

File: docs/api_reference.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.. _api_reference:
+
+API Reference
+=============
+
+
+This section documents sofar SOFA objects and functions. For examples on how to
+use sofar refer to the
+:ref:`sofa and SOFA <gallery:/gallery/interactive/sofar_introduction.ipynb>`
```
Comment (ahms5) [inline docs/api_reference.rst:9]: sofar and SOFA?

**Review by ahms5 — COMMENTED** [review]
Nice, thank you!

**Review by hoyer-a — APPROVED** [review]
Thanks! 
I think there is a new introduction coming from develop sofastream anyways, right?

**Comment by f-brinkmann** [comment]
Yes, we will have to resolve a conflict there, probably

**Pull request merged**